MIT.md

```diff
@@ -0,0 +1,8 @@
+## Copyright 2021, Daniel M. Matongo
+
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this website and associated documentation files, to deal in the website without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the website, and to permit persons to whom the website is furnished to do so, subject to the following conditions:
```

Comment:
- [ ] On the readme the live link there doesn't show this project. Kindly update it.

Reply:
Thanks its been handled

Reply:
- [ ] [OPTIONAL] Kindly update your live demo or check your deployment